pytest_fixtures.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+@pytest.fixture(scope='module')
+def db_url():
+    """ Database URL used in sqlalchemy.create_engine
+
+    Override this in your test to provide your desired url.
+    For valid urls see: http://docs.sqlalchemy.org/en/latest/core/engines.html
+    Defaults to SQLite memory database.
```

Comment:
probably worth a warning to make sure this is a _test_ db, since these fixtures will drop tables